### PR TITLE
feat: publisher-specific configuration editing

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of Changes

Base work for publisher-specific configuration editing.  Local API configuration addonbar tab added.

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

### CODE REVIEW

**Comment by GhostEagle68** [comment]
Wouldn't it look better to have save and discard at the bottom?

File: src/renderer/components/AircraftSection/Properties/index.tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/AircraftSection/Properties/index.tsx:2]: 2:10  error  'Link' is defined but never used       @typescript-eslint/no-unused-vars

File: src/renderer/components/AircraftSection/Properties/index.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+import { Publisher } from 'renderer/utils/InstallerConfiguration';
+import fs from 'fs';
+import path from 'path';
+import { Directories } from 'renderer/utils/Directories';
+import { Toggle } from '@flybywiresim/react-components';
+import { ArrowLeft } from 'react-bootstrap-icons';
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/AircraftSection/Properties/index.tsx:8]: 8:10  error  'ArrowLeft' is defined but never used  @typescript-eslint/no-unused-vars

**Review by FoxtrotSierra6829 — CHANGES_REQUESTED** [review]

File: src/renderer/components/Modal/index.tsx [hunk 3]
```diff
@@ -38,14 +38,14 @@ export const ModalProvider: FC = ({ children }) => {
 
 interface BaseModalProps {
     title: string;
-    bodyText: string;
+    bodyText?: string;
```
Comment (FoxtrotSierra6829) [inline src/renderer/components/Modal/index.tsx:41]: Why should this be optional?

**Comment by FoxtrotSierra6829** [comment]
grafik
Buttons don't match the other button's design theme

**Pull request closed without merging**